README.md

```diff
@@ -11,6 +11,19 @@ You can learn more about it in [the specification](http://bsonspec.org).
 - [Documentation](#documentation)
 - [FAQ](#faq)
 
+
+### Release Integrity
+
+The GitHub release contains a detached signature file for the NPM package (named
+`mongodb-X.Y.Z.tgz.sig`).
+
+To verify the integrity of the downloaded package, run the following command:
+
+```shell
+gpg --verify bson-X.Y.Z.tgz.sig bson-X.Y.Z.tgz
```

Comment:
Just a reminder to update these instructions when we have the final `.sig` file name and  how to get the tgz.

Reply:
Yep! I've updated them now, let me know your thoughts.